### PR TITLE
Post Page implementation Emberify + Redesign

### LINKED ISSUE #32: Implement Post Page
## Description

Implement the main "post" page. If you click on one of the articles in the index page you will be directed to the post page that shows the full content of a post.

Note: this page does not have a defined design for it. You should be able to implement it using the styleguide documentation (see link below). If you feel like you need some guidance please feel free to ask us in the comments on this issue, or by reaching out to us on the #st-website channel on Discord. 

If you would like to work on this issue you can "claim" it by commenting and letting us know that you would like to work on it 👍 

## Supporting information - please read if you are new to this project

This redesign effort is powered by the new version of the Ember Styleguide. You should be able to implement everything you need for this feature by following the [new documentation for the ember-styleguide](https://deploy-preview-145--ember-styleguide.netlify.com/) but if you think there is not enough documentation please let us know and we will point you in the right direction.

In most cases you should not need any CSS to implement this issue for the blog template, you should instead be trying to make use of the helpers and components that are documented in the ember-styleguide. This is not a strict rule, there may be cases where the documentation in the ember-styleguide is lacking or some custom CSS is actually needed. If you have any questions please comment on this issue or reach out to us on the `#st-website` channel on the [Ember Community Discord](https://discord.gg/emberjs)

### PR DESCRIPTION
**Pull request by ankushdharkar**
Fixes #32 

Without a design, tried to match the new [styleguide](https://ember-styleguide.netlify.app/) with an [example](https://blog.emberjs.com/2018/10/26/the-ember-times-issue-70.html)

### CODE REVIEW

File: app/templates/post.hbs [hunk 1]
```diff
@@ -1,35 +1,32 @@
-
 {{#let @model.post as |post|}}
 <article class="post-content {{unless post.image 'no-image'}}">
-
-    <header class="post-content-header">
-        <h1 class="post-content-title">{{post.title}}</h1>
-    </header>
-
+    {{!-- 
     {{#if post.image}}
```
Comment (ankushdharkar) [inline app/templates/post.hbs:4]: Still need to implement this

Comment (mansona) [reply app/templates/post.hbs:4]: Actually I was going to mention that we don't currently support "cover images" for posts so we can probably just delete it for now 👍

Comment (ankushdharkar) [reply app/templates/post.hbs:4]: Awesome. Will do. Thanks

File: addon/styles/post.css [hunk 2]
```diff
@@ -0,0 +1,45 @@
+.post-content-header h1 {
```
Comment (mansona) [inline addon/styles/post.css:1]: can't we use the css helpers for font size here anyway? 🤔 https://ember-styleguide.netlify.app/concepts/typography

Comment (mansona) [inline addon/styles/post.css:1]: can't we use the css helpers for font size here anyway? 🤔 https://ember-styleguide.netlify.app/concepts/typography

Comment (mansona) [inline addon/styles/post.css:1]: can't we use the css helpers for font size here anyway? 🤔 https://ember-styleguide.netlify.app/concepts/typography

Comment (mansona) [inline addon/styles/post.css:1]: can't we use the css helpers for font size here anyway? 🤔 https://ember-styleguide.netlify.app/concepts/typography

**Comment by mansona** [comment]
Hey sorry I had an in-progress review that I never completed 🙈 I submitted it there now and added a bit more in a second review. Let me know if you have any questions and I'll take another look once this is finished 👍

**Comment by ankushdharkar** [comment]
No worries. Will get to the changes this week. Thank you

**Review by mansona — APPROVED** [review]
Great work 🎉  thank you both for working on this!

**Pull request merged**